assets/cross-sell-carousel.css

```diff
@@ -0,0 +1,88 @@
+@media (min-width: 751), (max-width: 2000px) {
+
+  .cross-sell-product-title {
+    /* show only 2 lines of text */
+    display: block;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
+    overflow: hidden;
+    max-height: 3.6em;
+    line-height: 1.8em;
+    /* end */
+    padding: 0 10px;
+    font-size: 12px;
+  }
+
+  .glide__arrow--left {
```

Comment:
This is dangerously unspecific, you have two elements on this page with this exact class.